setup.py

```diff
@@ -24,14 +23,28 @@
 ]
 
 authors = [
-    "TODO"
+    "Bhavani Iyer <bsiyer@us.ibm.com>",
+    "Avirup Sil <avi@us.ibm.com>",
+    "Martin Franz <franzm@us.ibm.com>",
+    "Mihaela Bornea <mabornea@us.ibm.com>",
+    "Sara Rosenthal <sjrosenthal@us.ibm.com>",
+    "Avirup Sil <avi@us.ibm.com>",
+    "Scott McCarley <jsmc@us.ibm.com>",
+    "Rong Zhang <zhangr@us.ibm.com>",
+    "Jaydeep Sen <jaydesen@in.ibm.com>",
+    "Yulong Li <yulongli@us.ibm.com>",
+    "Md. Arafat Sultan <Arafat.Sultan@ibm.com>",
+    "Vishwajeet Kumar024 <vishk024@in.ibm.com>",
+    "Saneem A Chemmengath <saneem.cg@in.ibm.com>",
+    "Anthony Ferritto",
 ]
 
 _deps = {
     "docutils>=0.14,<0.18": ["tests"],
     "bitarray~=2.3.7": ["install"],
     "bump2version~=1.0.1": ["dev"],
     "click~=8.0.4": ["install"],
+    "cupy-cuda113~=10.4.0": ["install"],
```

Comment:
Was this necessary for something other than ColBERT?

Reply:
I don't think so. @franzm @bhavani105 @jdpsen @jsmcibm ?

Reply:
The setup.py needs to be synched with the version in main.

Reply:
not in QG or QA.